js homework week2/file10.js

```diff
@@ -0,0 +1,15 @@
+let myArray = [true , 5 , 'yigit'];
+console.log(myArray);
+console.log('Here is an array that includes 3 types of element. They are boolean, number and string ')
+console.log(typeof myArray);
+console.log(typeof myArray[0]);
+console.log(typeof myArray[1]);
+console.log(typeof myArray[2]);
+
+
+
+
+let infinity1 = 6/0;
+let infinity2 = 10/0;
+if (infinity1===infinity2){console.log(true)}

```

Comment:
The solution is good! But your code is easier to read if you write statements like this on multiple lines (and add more spaces). For example:

```
if (infinity1 === infinity2){
  console.log(true);
} else {
  console.log(false); 
}
```

Reply:
ok done